src/OrbitBase/Logging.cpp

```diff
@@ -35,22 +52,34 @@ ErrorMessageOr<void> TryRemoveOldLogFiles(const std::filesystem::path& log_dir)
   return orbit_base_internal::RemoveFiles(old_files);
 }
 
-void InitLogFile(const std::filesystem::path& path) {
+void InitLogFile(const std::filesystem::path& path) noexcept {
   absl::MutexLock lock(&log_file_mutex);
   // Do not call CHECK here - it will end up calling LogToFile,
   // which tries to lock on the same mutex a second time. This will
   // lead to an error since the mutex is not recursive.
-  if (log_file.is_open()) {
+  if (log_file != nullptr) {
     PLATFORM_ABORT();
   }
-  log_file.open(path, std::ofstream::out);
+
+  // TEMP_FAILURE_RETRY for FILE*
+  do {
+    log_file.reset(
+        fopen(path.c_str(), "wbe"));  // O_WRONLY, O_CLOEXEC for glibc, O_BINARY for windows
```

Comment:
On Windows  'e' in "wbe" gives me "Invalid file open mode".

Reply:
yep , add ifdef guard here too.